orchestration/dagster_orchestration/hca_orchestration/solids.py

```diff
@@ -59,3 +60,16 @@ def submit_file_ingest(context) -> Nothing:
     """
     datasets = context.resources.data_repo_client.enumerate_datasets()
     context.log.debug(f"Enumerate found {datasets.total} datasets in the repo")
+
+
+@solid(
+    config_schema={"gcp_env": Field(str, is_required=False, default_value="dev")},
```

Comment:
❓ I'm curious why this shouldn't be required; defaulting to `dev` seems like it opens us up to inadvertently running things in dev when we meant another environment like prod

Reply:
Good point; will change